### ISSUE TITLE
Closing last tab does not reopen user defined home page

### ISSUE THREAD

**Issue by xaxousis** [bwinton/TabCenter#915]
Firefox is configured to not close the window after the last tab is closed. Closing the last tab reopens the *New Tab* page instead of the user defined home page.

### Environment
Linux (Debian stable)
Firefox 51.0.1

### Steps to reproduce
  - set `browser.tabs.closeWindowWithLastTab` to `false`.
  - set home page in preferences
  - close all tabs

### Expected result
After the last tab is closed, a new one is opened with the home page.

### Actual result
After the last tab is closed, the *New Tab* page is opened.

### Notes
  - Refreshing the page opens the correct home page.
  - If the tabs are moved to the top again, the bug disappears.

**Comment by SoftVision-CiprianMuresan** [comment]
Hi @xaxousis,

I've followed your steps to reproduce without Tab Center installed and the same thing occurs, when I close the last tab, the New Tab page is opened and refreshing the page just refreshes the New Tab page. The same thing happens with Tab Center installed, so this leads me to believe that it's not a Tab Center issue.

Did you set your homepage from about:preferences? Or did you edit a pref? (if yes, which one?)

Could you please disable all other add-ons except Tab Center and see if you can reproduce the issue?

**Comment by xaxousis** [comment]
Hi @SoftVision-CiprianMuresan,

I tried with all modules disabled and I get the same result as you.

It seems to be the [Speed Dial [FVD] - New Tab Page, Sync...](https://addons.mozilla.org/fr/firefox/addon/fvd-speed-dial/?src=userprofile) module that triggers the behaviour I believed was correct when its option *show in last tab* (translated from French) is checked.

I had checked without disabling the modules but changing the homepage and though it had worked, which was a mistake. 

The homepage was set from about:preferences.

This said, I believe you are right. :)
